docs/reference/esql/processing-commands/keep.asciidoc

```diff
@@ -40,10 +50,84 @@ include::{esql-specs}/docs.csv-spec[tag=keepWildcard]
 ----
 
 The asterisk wildcard (`*`) by itself translates to all columns that do not
-match the other arguments. This query will first return all columns with a name
+match the other arguments.
+
+This query will first return all columns with a name
 that starts with `h`, followed by all other columns:
 
 [source,esql]
 ----
 include::{esql-specs}/docs.csv-spec[tag=keepDoubleWildcard]
 ----
+
+The following examples show how precedence rules work when a field name matches multiple expressions.
+
+Full name duplicates, last one wins:
+
+[source,esql]
+----
+include::{esql-specs}/docs.csv-spec[tag=keepMultipleMatch]
+----
+[%header.monospaced.styled,format=dsv,separator=|]
+|===
+include::{esql-specs}/docs.csv-spec[tag=keep-multiple-match-result]
+|===
```

Comment:
Personal opinion: that's quite a few examples. Maybe the full name duplicate one could be removed, as the "last one wins" logic is already illustrated below in `keepWildcardPrecedence`?

Reply:
Yeah I agree, was erring on the side of verbosity in first draft, also was wrestling with tests more than actual content, but I think I've got testing tamed now 😄